packages/freetds/freetds.0.4/opam

```diff
@@ -20,3 +26,10 @@ build: [
 remove: [["ocamlfind" "remove" "freetds"]]
 depends: ["ocamlfind"]
 install: [make "install"]
+depexts: [
+  [["centos"] ["autoconf" "automake" "freetds-devel"]]
+  [["debian"] ["autoconf" "automake" "freetds-dev"]]
+  [["fedora"] ["autoconf" "automake" "freetds-devel"]]
+  [["ubuntu"] ["autoconf" "automake" "freetds-dev"]]
+  [["osx" "homebrew"] ["autoconf" "automake" "freetds"]]
```

Comment:
you might maybe want to depend on the `conf-autoconf` package instead. It would avoid redundancies.